src/plugins/telemetryTable/TelemetryTable.js

```diff
@@ -372,6 +376,9 @@ define([
         this.domainObject.configuration.filters &&
         this.domainObject.configuration.filters[keyString];
 
+      // de-reactify it
+      filters = toRaw(filters);
```

Comment:
We might not want to couple this with Vue, is there a way we could ensure the filters are not wrapped in a proxy before being persisted on the config?

Reply:
Great point. No reason this should be in here, I'll see if it's even needed. The others in the Vue file may be enough.